src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/GregtechMetaTileEntity_QuantumForceTransformer.java

```diff
@@ -712,9 +712,10 @@ public boolean checkRecipe(final ItemStack aStack) {
 
     private boolean processRecipe(
             ItemStack[] aItemInputs, FluidStack[] aFluidInputs, GT_Recipe.GT_Recipe_Map aRecipeMap, ItemStack aStack) {
-        long tVoltage =
-                getMaxInputVoltage() / getExoticAndNormalEnergyHatchList().size();
-        long tAmps = (long) Math.floor(getMaxInputAmps() * 0.80);
+        int hatches = getExoticAndNormalEnergyHatchList().size();
+        long tVoltage = getMaxInputVoltage() / hatches;
+        // Need to check weather the hatches used are TT ones or not as TT hatches can request 20% more amps
+        long tAmps = (long) Math.floor(mExoticEnergyHatches.isEmpty() ? getMaxInputAmps() : getMaxInputAmps() * 0.80);
```

Comment:
Wrong way around ? (it's the opposite way than the comment says or the comment should say less) . Also why would you cut off 20% of usable power ?

Reply:
a 4A energy hatch from TT can accept 5A, which gets added to max Input Amps. a 64A hatch can accept up to 80A. (so technically it accepts 25% more and i want to remove it so i remove 20%)

Reply:
Ah ok, I misunderstood the comment

Reply:
it gets worse with lasers as a 256A laser is counted as a 320A hatch